app/src/main/java/com/team33/qrcodepursuit/MainActivity.java

```diff
@@ -1,13 +1,45 @@
 package com.team33.qrcodepursuit;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+
 import android.os.Bundle;
+import android.view.MenuItem;
+
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import com.google.android.material.navigation.NavigationBarView;
 
 public class MainActivity extends AppCompatActivity {
 
+    private BottomNavigationView menu;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        menu = findViewById(R.id.bottomnavigation);
+
+        menu.setOnItemSelectedListener(new NavigationBarView.OnItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+
+                Fragment frag = null;
+
+                switch (item.getItemId()) {
```

Comment:
so basically this would be the thing switching between home/map/leaderboard/profile right?
```
switch ( item.getId() )
    case R.id.bottomnavigation_menu_scan:
        // switch to scan
    case R.id.bottomnavigation_menu_map:
        // switch to map 
    case R.id.bottomnavigation_menu_leaderbd:
        // switch to leaderboard
    case R.id.bottomnavigation_menu_profile:
        // switch to profile
    default:
         break
```

Reply:
yee